package.json

```diff
@@ -26,6 +26,7 @@
     "moment-timezone": "^0.5.33",
     "vue-barcode-reader": "^1.0.1",
     "vue": "^3.2.26",
+    "vue-barcode-reader": "^1.0.1",
```

Comment:
I think this is added twice. Check line 27

Reply:
Reverted this change.